website/content/docs/security/encryption.mdx

```diff
@@ -1,10 +1,8 @@
 ---
 layout: docs
-page_title: Encryption
+page_title: Encryption Systems
 description: >-
-  The Consul agent supports encrypting all of its network traffic. The exact
-  method of encryption is described on the encryption internals page. There are
-  two separate encryption systems, one for gossip traffic and one for RPC.
+Two encryption systems protect Consul’s network communications. Learn how keys secure gossip between agents and how RPC with TLS certificates verifies the authenticity of servers and clients.
```

Comment:
Switching the order, since RPC communication is universally relevant (even on runtimes that have removed gossip). I also like the first sentence from the original description that makes it clear that all network traffic can be encrypted.

```suggestion
Consul supports encrypting all of its network traffic. Remote Process Calls (RPCs) between client and server agents can be encrypted with TLS and authenticated with certificates. Gossip communication between all agents can also be encrypted.
```